src/sqlstore/champion.py

```diff
@@ -0,0 +1,120 @@
+from sqlalchemy import Column, Integer, String, Float, Text, CheckConstraint, PickleType, DateTime
+from sqlalchemy.sql import func
+from src.sqlstore.db import Base
+
+
+class SQLChampion(Base):
+    __tablename__ = "champion"
+
+    championId = Column(Integer, primary_key=True)
+    patchNumber = Column(Integer, primary_key=True)
+    championName = Column(String(100))
+    championTitle = Column(String(100))
+    infoAttack = Column(Integer)
+    infoDefense = Column(Integer)
+    infoMagic = Column(Integer)
+    infoDifficulty = Column(Integer)
+    tags = Column(PickleType)   # serialized list of tags (eg. [Marksman, Support] for Ashe)
+    partype = Column(String(150))   # type of mana or energy (eg. "Blood Well" for Aatrox)
+    patchWinRate = Column(Float, nullable=True)  # Represented as a percent
+    patchPlayRate = Column(Float, nullable=True)  # Represented as a percent
+    # TODO: should this be part of the primary key?
+    primaryRole = Column(String(50), nullable=True)  # Top, Mid...

```

Comment:
Not including primaryRole in the primary key to maintain flexibility. A champion may have multiple roles in a given patch. (Warwick top...)